examples/ntp_simpletest.py

```diff
@@ -0,0 +1,46 @@
+import time
+import board
+import busio
+from digitalio import DigitalInOut
+from adafruit_esp32spi import adafruit_esp32spi
+from adafruit_ntp import NTP
+
+# If you are using a board with pre-defined ESP32 Pins:
+esp32_cs = DigitalInOut(board.ESP_CS)
+esp32_ready = DigitalInOut(board.ESP_BUSY)
+esp32_reset = DigitalInOut(board.ESP_RESET)
+
+# If you have an externally connected ESP32:
+# esp32_cs = DigitalInOut(board.D9)
+# esp32_ready = DigitalInOut(board.D10)
+# esp32_reset = DigitalInOut(board.D5)
+
+spi = busio.SPI(board.SCK, board.MOSI, board.MISO)
+esp = adafruit_esp32spi.ESP_SPIcontrol(spi, esp32_cs, esp32_ready, esp32_reset)
+
+print("Connecting to AP...")
+while not esp.is_connected:
+    try:
+        esp.connect_AP(b"WIFI_SSID", b"WIFI_PASS")
+    except RuntimeError as e:
+        print("could not connect to AP, retrying: ", e)
+        continue
+
+# Initialize the NTP object
+ntp = NTP(esp)
+
+# Fetch and set the microcontroller's current UTC time
+ntp.set_time()
+
+# Get the current time in seconds since Jan 1, 1970
+current_time = time.time()
+print("Seconds since Jan 1, 1970: {} seconds".format(current_time))
+
+# Convert the time in seconds to a struct_time
+now = time.localtime(current_time)
```

Comment:
as above "local time" is confusing.  are all times in UTC? will they always be?

Reply:
`localtime` is a `time_struct` reflecting the local time, not UTC. I'll update the PR to reflect this.

https://circuitpython.readthedocs.io/en/latest/shared-bindings/time/__init__.html#time.localtime